pkg/cli/admin/release/extract_tools.go

```diff
@@ -465,11 +465,33 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			Arch:    targetReleaseArch,
 			Command: "ccoctl",
 			NewArch: true,
-			Mapping: extract.Mapping{Image: "cloud-credential-operator", From: "usr/bin/ccoctl"},
+			Mapping: extract.Mapping{Image: "cloud-credential-operator", From: "usr/bin/ccoctl.rhel8"},
```

Comment:
This would break the users whose are running this command against older release payloads that don't have `ccoctl.rhel8`. It makes sense to provide a mechanism to download rhel8 and rhel9 based ccoctls just like we did for oc. However, I think it is crucial to determine the behavior against old release payloads.